retry

```diff
@@ -6,24 +6,28 @@ options:
     -h,--help               Show this help
     -r,--retries=RETRIES    How many retries to do on command failure after the
                             initial try. Defaults to 3.
-    -s,--sleep=SLEEP        How many seconds to sleep between retries
+    -s,--sleep=SLEEP        How many seconds to sleep between retries. Defaults
+                            to 3 seconds.
+    -e,--exponential        Enable simple binary exponential back-off
+                            algorithm. Disabled by default.
```

Comment:
Awesome! I already thought about exponential back-off and you already implement it! :) Here I had the idea to not have a boolean variable but a factor defaulting to 1 and multiplying the sleep time. How about

```
-S,--sleep-factor=SLEEP_FACTOR …something about exponential back-off… Defaults to 1 meaning same sleep time between each try.
```

Reply:
I experimented with using the SLEEP as the base factor, but that did not feel good enough for larger sleep values so I decided to use the binary base and multiply that with SLEEP.